lib/marked.js

```diff
@@ -108,7 +108,7 @@ block.gfm.paragraph = edit(block.paragraph)
 
 block.tables = merge({}, block.gfm, {
   nptable: /^ *([^|\n ].*\|.*)\n *([-:]+ *\|[-| :]*)(?:\n((?:.*[^>\n ].*(?:\n|$))*)\n*|$)/,
-  table: /^ *\|(.+)\n *\|?( *[-:]+[-| :]*)(?:\n((?: *[^>\n ].*(?:\n|$))*)\n*|$)/
+  table: /^ *\|(.+)\n *\|?( *[-:]+[-| :]*)(?:\n((?: *[^>\n #(```)-].*(?:\n|$))*)\n*|$)/
```

Comment:
I don't think `(```)` works as you expect. This will prevent even a single `` ` `` or `(` or `)` from counting as a inline characters

```md
| abc | def |
| --- | --- |
| bar | foo |
| baz | boo |
`foobar()`
```
should be

| abc | def |
| --- | --- |
| bar | foo |
| baz | boo |
`foobar()`

and

```md
| abc | def |
| --- | --- |
| bar | foo |
| baz | boo |
(foobar())
```
should be

| abc | def |
| --- | --- |
| bar | foo |
| baz | boo |
(foobar())

Reply:
and a `-` and `#` could be inline characters

```md
| abc | def |
| --- | --- |
| bar | foo |
| baz | boo |
#foobar()
-foobar()
```

| abc | def |
| --- | --- |
| bar | foo |
| baz | boo |
#foobar()
-foobar()

Reply:
`>` is the only single character that can't be inline at the beginning of a line

```md
| abc | def |
| --- | --- |
| bar | foo |
| baz | boo |
>foobar()
```

| abc | def |
| --- | --- |
| bar | foo |
| baz | boo |
>foobar()